app/src/main/java/com/example/hcm_102_0006/android_project_m/view/adapter/AdapterGenres.java

```diff
@@ -0,0 +1,73 @@
+package com.example.hcm_102_0006.android_project_m.view.adapter;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.databinding.DataBindingUtil;
+import android.databinding.ObservableField;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.hcm_102_0006.android_project_m.R;
+import com.example.hcm_102_0006.android_project_m.service.model.Genres;
+import com.example.hcm_102_0006.android_project_m.databinding.ItemGenres2Binding;
+
+import java.util.List;
+
+/**
+ * Created by hcm-102-0006 on 22/11/2017.
+ */
+
+public class AdapterGenres extends RecyclerView.Adapter<AdapterGenres.MyViewHolder> {
+
+    private Activity mContext;
+    public static final String KEY_RESULT = "GENRES";
+    private List<Genres> mGenres;
+    public AdapterGenres(List<Genres> mGenres,Activity context) {
```

Comment:
2.9 Parameter ordering in methods

In Android, it is common to define methods that take a Context. If you write a method, Context should be the first parameter. And callback always should be the last parameter.